public/css/styles.css

```diff
@@ -53,7 +90,8 @@ button:disabled {
 .centered-form {
   display: flex;
   align-items: center;
-  height: 100vh;
```

Comment:
This is a kind of work around to make it looking nice on mobile devices. I did not find a good solution for the problem with mobile devices. Maybe later I will come to this again...